packages/fluxible-plugin-devtools/src/lib/devtools-plugin.js

```diff
@@ -155,7 +159,7 @@ export default function devToolsPlugin() {
 
                     return {
                         actionContext: actionContext,
-                        action: timedAction,
+                        action: (action.length < 3) ? timedActionNoCallback : timedAction,
```

Comment:
timedAction does a return on the wrapped action, and I do a return on timedAction, so if a promise is returned it would be returned